crates/iota-transactional-test-runner/src/test_adapter.rs

```diff
@@ -1337,10 +1392,13 @@ impl IotaTestAdapter {
         };
         let aa_sender_addr = aa_arg.id().into();
 
+        let gas_ref = self
+            .fund_address_for_testing(aa_sender_addr, 100_000_000_000_000)
```

Comment:
```suggestion
            .fund_address_for_testing(aa_sender_addr, GAS_FOR_TESTING)
```
There is already a constant defined for this scope. Furthermore, in any case, constants need to be declared in the top section of the file.

Reply:
GAS_FOR_TESTING is not enough we will encounter - `Insufficient coin balance for operation`
I'll create another const for AA gas case.

Reply:
Fixed cba43a003caf5a796bbccaeff46d3215d3dc84f5